### PR TITLE
HPCC-15248 Allow daliadmin to display stats from multiple workunits

### PR DESCRIPTION
**Pull request by ghalliday**
Signed-off-by: Gavin Halliday gavin.halliday@lexisnexis.com

### CODE REVIEW

**Comment by ghalliday** [comment]
@jakesmith please review

File: dali/daliadmin/daliadmin.cpp [hunk 1]
```diff
@@ -2458,6 +2454,37 @@ static void dumpStats(const char *wuid, const char * creatorTypeText, const char
...
+    Owned<IWorkUnitFactory> factory = getWorkUnitFactory();
+    const char * star = strchr(wuid, '*');
+    if (star)
+    {
+        WUSortField filters[3];
+        MemoryBuffer filterbuf;
+        filters[0] = WUSFwuid;
+        filterbuf.append(star-wuid, wuid).append((char)0);
+        filters[1] = WUSFwuidhigh;
+        filterbuf.append(star-wuid, wuid).append(127).append((char)0);
```
Comment (jakesmith) [inline dali/daliadmin/daliadmin.cpp]: I think WUSFwildwuid is more appropriate?

**Comment by jakesmith** [comment]
@ghalliday - 1 question/comment.

**Comment by ghalliday** [comment]
@jakesmith thanks, that is much cleaner.
@richardkchapman I think this is ready.

**Pull request merged**